.github/workflows/test.yml

```diff
@@ -1,4 +1,4 @@
-name: build-and-test
+name: test
```

Comment:
My apologies, I think build-and-test would be better than test for both. It just seems closer to what's actually happening.

Reply:
```suggestion
name: build-and-test
```

Reply:
I'll rename the file.